LICENSE

```diff
@@ -1,6 +1,4 @@
-The MIT License (MIT)
-
-Copyright (c) 2015 SiLab
+Copyright (c) 2018 SiLab, University of Bonn
```

Comment:
Copyright information should not be updated since they state the date of your first copyright claim.